onnxruntime/test/python/onnx_backend_test_series.py

```diff
@@ -124,6 +124,14 @@ def create_backend_test(testname=None):
                                  '^test_range_float_type_positive_delta_expanded_cpu.*',
                                  '^test_range_int32_type_negative_delta_cpu.*',
                                  '^test_range_int32_type_negative_delta_expanded_cpu.*',
+                                 '^test_clip_cpu.*',
```

Comment:
why change clip tests in scatter_nd change?

Reply:
this seems to be outdated. I guess it's a result of multiple rebase that has happened. I couldn't find this line of change in the "Files changed" tab.